web/src/main/webapp/kew/WEB-INF/web.xml

```diff
@@ -1,220 +1,219 @@
-<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
In most of the web.xml files, the only change was to use web-app_2.5 instead of web-app_2.4